app/pubsub-listener.js

```diff
@@ -147,7 +147,9 @@ export default class PubsubListener {
         return;
       }
 
-      if (await post.isBannedFor(user.id)) {
+      const banIds = await user.getBanIds()
+
+      if (banIds.indexOf(post.userId) >= 0) {
```

Comment:
it is. but the idea is, that I can reuse `banIds` a bit later in this code, while `isBannedFor` hid the data from me